canonical-web-and-design/jaas-dashboard
# Convert Banner to Typescript

## Done

Convert Banner to Typescript

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check all of the things pass
